sematic/ui/packages/main/src/runs/RunIndex.tsx

```diff
@@ -163,6 +165,16 @@ export function RunIndex() {
         [searchString]
     );
 
+    const [submitedDefault, setSubmitedDefault] = useState(false);
+
+    useEffect(() => {
+        if(submitedDefault || !defaultSearchString) {
```

Comment:
Simply read the deep link value from the hash here, then set both `setSubmitedSearchString` and `setSearchString` should work. For one-off operation, the dependency array can be empty.